### PR TITLE
SO-5730: Port 7.x changes to the 8.x stream

### PR DESCRIPTION
**Pull request by AAAlinaaa**

### CODE REVIEW

**Comment by apeteri** [comment]
The build passes both locally and in GH's runner.

File: commons/com.b2international.index/lib/elasticsearch-log4j-7.16.3.jar
Comment (apeteri) [inline commons/com.b2international.index/lib/elasticsearch-log4j-7.16.3.jar:1]: We don't need this version as a customized 7.17.2 JAR already exists on 8.x.

File: commons/com.b2international.index.tests/src/com/b2international/index/mapping/MappingMigrationTest.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 import java.util.Collection;
 import java.util.List;
+import java.util.Map;
```
Comment (apeteri) [inline commons/com.b2international.index.tests/src/com/b2international/index/mapping/MappingMigrationTest.java]: `Map` is not used in this class.

File: core/com.b2international.snowowl.core.rest.tests/src/com/b2international/snowowl/core/rest/AllSnowOwlApiTests.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2011-2023 B2i Healthcare Pte Ltd, http://b2i.sg
+ * Copyright 2011-2022 B2i Healthcare Pte Ltd, http://b2i.sg
```
Comment (apeteri) [inline core/com.b2international.snowowl.core.rest.tests/src/com/b2international/snowowl/core/rest/AllSnowOwlApiTests.java]: The copyright notice has doubled up.

File: snomed/com.b2international.snowowl.snomed.common/src/com/b2international/snowowl/snomed/common/SnomedConstants.java [hunk 3]
```diff
@@ -119,6 +119,7 @@ private Concepts() { }
 		public static final String REFSET_DESCRIPTION_TYPE = "900000000000538005";
 		public static final String REFSET_CONCRETE_DOMAIN_TYPE_AU = "50131000036100"; //AU release -> NEHTA_0856_2012_AMTImplentationKit_20120229
 		public static final String REFSET_MODULE_DEPENDENCY_TYPE = "900000000000534007";
+		public static final String REFSET_SIMPLE_MAP_TO_SNOMEDCT = "1187636009";
```
Comment (apeteri) [inline snomed/com.b2international.snowowl.snomed.common/src/com/b2international/snowowl/snomed/common/SnomedConstants.java]: Constant is duplicated on line 118.

**Review by apeteri — APPROVED** [review]
LGTM, great work! I only have small issues that need to be fixed before merging.

**Review by nagyo — APPROVED** [review]
RSLGTM!

**Pull request merged**